docs/decisions/003-additional-metrics-fields.md

```diff
@@ -0,0 +1,42 @@
+# DR: Adding additional fields to the Anonymous Metrics
+
+## Context
+
+Spice reports certain metrics anonymously, such as query count, bytes processed, bytes returned, query duration,
```

Comment:
The context doesn't address why this DR was created. Why are we pondering this decision?